docs/notary.md

```diff
@@ -172,7 +172,7 @@ Notary contract and designate `P2PNotary` node role in RoleManagement native
 contract.
 
 If you use custom `NativeActivations` subsection of the `ProtocolConfiguration`
-section in your node config, then specify the height of the Notary contract
+section in your node config, it specifies the height of the Notary contract
```

Comment:
Not exactly. If `ProtocolConfiguration` is explicitly set, then you MUST specify the height for Notary, otherwise it won't work.